src/main/kotlin/de/bund/digitalservice/useid/refresh/RefreshController.kt

```diff
@@ -38,8 +36,7 @@ class RefreshController(private val identificationSessionService: Identification
         @RequestParam requestQueryParams: Map<String, String>,
     ): ResponseEntity<Unit> {
         val session = identificationSessionService.findByEIdSessionId(eIdSessionId) ?: run {
-            log.error("Failed to load identification session.")
-            return ResponseEntity.status(HttpStatus.NOT_FOUND).build()
+            throw IdentificationSessionNotFoundException()
```

Comment:
We don't need that `run` thing and can just use `?: throw ...`

Reply:
Yes. Resolved by moving this to the `findByEIdSessionId` method (see comment above).